src/evict/evict_lru.c

```diff
@@ -188,6 +184,29 @@ __wt_evict_list_clear_page(WT_SESSION_IMPL *session, WT_REF *ref)
         __wt_spin_unlock(session, &cache->evict_queues[q].evict_lock);
     }
     WT_ASSERT(session, !F_ISSET_ATOMIC_16(ref->page, WT_PAGE_EVICT_LRU));
+}
+
+/*
+ * __wt_evict_list_clear_page --
+ *     Make sure a page is not in the LRU eviction list. This is called from the page eviction code
```

Comment:
Instead of "Make sure", I feel that the header of `__evict_list_clear_page_locked` is more appropriate here.

This function checks for all LRU lists, excluding the urgent queue and removes the page if found.

Reply:
Sorry, could you elaborate.

Reply:
Updated in [b3dae3d](https://github.com/wiredtiger/wiredtiger/pull/8950/commits/b3dae3d8dfb7004b78e44448c245548d88c340a7)